infra/src/main/scala/scaladex/infra/storage/sql/tables/ProjectTable.scala

```diff
@@ -64,12 +61,12 @@ object ProjectTable {
     buildSelect(
       fullTableFr,
       allFieldsFr,
-      fr0"WHERE creation_date IS NOT NULL ORDER BY creation_date DESC LIMIT $limit"
+      fr0"WHERE p.creation_date IS NOT NULL ORDER BY p.creation_date DESC LIMIT ${limit.toLong}"
```

Comment:
Is this `limit.toLong` needed because of doobie check?

```suggestion
      fr0"WHERE p.creation_date IS NOT NULL ORDER BY p.creation_date DESC LIMIT ${limit.toLong}"
```

Reply:
yes